ports/matio-cpp/portfile.cmake

```
@@ -0,0 +1,30 @@
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO ami-iit/matio-cpp
+    REF "${VERSION}"
+    SHA512 efae9cec18b44291625f0770dabe8fd23c5b1d7cf77849cab827880ca96328430d4146014ace82d443bbc00cc313fd23cd5d67a7f1b58235efa7b5cb291a8ea4
+    HEAD_REF master
+    PATCHES
+    fix-dependencies.patch
+)
+
+file(COPY "${CMAKE_CURRENT_LIST_DIR}/matioCppConfig.cmake.in" DESTINATION "${SOURCE_PATH}")
+
+vcpkg_find_acquire_program(PKGCONFIG)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    MAYBE_UNUSED_VARIABLES
+        PKG_CONFIG_EXECUTABLE
```

Tell CMake where to find pkg-config.
```suggestion
    OPTIONS
        "-DPKG_CONFIG_EXECUTABLE=${PKGCONFIG}"
```